apps/verticalhome/test/unit/app_test.js

```diff
@@ -76,6 +76,15 @@ suite('app.js > ', function() {
     window.dispatchEvent(new CustomEvent('context-menu-close'));
   });
 
+  test('Transition property in edit-mode', function() {
+    app.grid.dispatchEvent(new CustomEvent('editmode-start'));
+    assert.equal(app.grid.style.transition,
+      'transform 0.25s ease 0s, height 0s ease 0.5s');
```

Comment:
If the style string was a property on the object, it could be referenced here, rather than duplicating it (which would be prone to breakage).